backend/app/routes/dummy/emailservice.js

```diff
@@ -0,0 +1,11 @@
+const sendEmail = require('../../../helpers/mailer');
+
+module.exports = {
+    sendVerificationEmail
+}
+async function sendVerificationEmail(email) {
```

Comment:
this should be `sendEmail` instead of `sendVerificationEmail`

Reply:
It should be sendVerificationEmail. I think you've misunderstood the code. The sendVerificationEmail function consists of sendEmail function and takes parameters.